src/Event/ReadModel/RDF/RdfProjector.php

```diff
@@ -131,6 +137,10 @@ public function handle(DomainMessage $domainMessage): void
 
         $this->setTerms($resource, $event->getTerms());
 
+        if ($event->getOrganizerReference()) {
+            $this->setOrganizer($resource, $event->getOrganizerReference());
+        }
+
         (new WorkflowStatusEditor())->setWorkflowStatus($resource, $event->getWorkflowStatus());
         if ($event->getAvailableFrom()) {
             (new WorkflowStatusEditor())->setAvailableFrom($resource, $event->getAvailableFrom());
```

Comment:
Is there a reason this WorkflowStatusEditor is not a dependency, or only created once?
Maybe you can make it once and then do 2 calls on it?

Reply:
There will be only one implementation of the `WorkflowStatusEditor`, another approach could have been a `trait`. But I will store it inside a local variable.